web/src/components/TaskPageSwitchComponent.test.tsx

```diff
@@ -0,0 +1,92 @@
+import { TaskPageSwitchComponent } from "@/components/TaskPageSwitchComponent";
+import { generateRoadmap, generateTask } from "@/test/factories";
+import { generateBusiness } from "@businessnjgovnavigator/shared/test";
+import { render, screen } from "@testing-library/react";
+
+describe("TaskPageSwitchComponent", () => {
+  const initialFeatureCigaretteLicense = process.env.FEATURE_CIGARETTE_LICENSE;
+
+  afterEach(() => {
+    process.env.FEATURE_CIGARETTE_LICENSE = initialFeatureCigaretteLicense;
+  });
+
+  describe("FEATURE_CIGARETTE_LICENSE feature flag", () => {
+    const task = generateTask({ id: "cigarette-license" });
+    const roadmap = generateRoadmap({});
+    const business = generateBusiness({});
+
+    const displayContent = {
+      formationDbaContent: {
+        DbaResolution: {
+          id: "",
+          name: "",
+          urlSlug: "",
+          callToActionLink: "",
+          callToActionText: "",
+          summaryDescriptionMd: "",
+          contentMd: "",
+          required: undefined,
+          agencyId: undefined,
+          agencyAdditionalContext: undefined,
+          formName: undefined,
+        },
+        Authorize: {
+          id: "",
+          name: "",
+          urlSlug: "",
+          callToActionLink: "",
+          callToActionText: "",
+          summaryDescriptionMd: "",
+          contentMd: "",
+          required: undefined,
+          agencyId: undefined,
+          agencyAdditionalContext: undefined,
+          formName: undefined,
+        },
+        Formation: {
+          id: "",
+          name: "",
+          urlSlug: "",
+          callToActionLink: "",
+          callToActionText: "",
+          summaryDescriptionMd: "",
+          contentMd: "",
+          required: undefined,
+          agencyId: undefined,
+          agencyAdditionalContext: undefined,
+          formName: undefined,
+        },
+      },
+    };
+
+    it("renders tax clearance certificate element when true", () => {
```

Comment:
[pebble] These tests are referencing "tax clearance certificate". I think we've been doing tax clearance work for too long. 😉